templates/beego/files/conf/app.conf

```diff
@@ -0,0 +1,4 @@
+appname = template-beego
+//httpaddr = "127.0.0.1"
+httpport = 8888
```

Comment:
Can this be made dynamic?  It's always 8888 on Psh right now, but it ought to be `$PORT`.

Reply:
I was trying to do that before when I was asking if PORT was available at build. My thinking:

- I tried `httpport = $PORT`, and when it didn't work I thought "Okay, .conf files can read env vars, I guess I should write this file on build."
- Tried doing that, PORT not available in build. 
- left it as hardcoded 8888

How else should I approach this?

Reply:
Does the author have a help channel, or project, or forum, or something where we can ask?  Doing it in a way that plays nice with the framework would be ideal.  If there isn't one, then I guess we hard code it with a comment about why it's being hard coded.